### PR TITLE
Update migrate-from-umarketingsuite.md

### PR DESCRIPTION
**Pull request by Jette94**
Added some changes to the guide after going through it

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks @Jette94 !

I've added some suggestions and questions.
Please take a look when you have a minute.

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md [hunk 1]
```diff
@@ -123,11 +127,11 @@ Based on the [Key Changes](migrate-from-umarketingsuite.md#key-changes) below up
 
 Please find below an overview of the changes to the default scripts in a uMarketingSuite installation:
 
-* All scripts & Asset Paths containing the `uMarketingSuite` keyword are renamed to:
+* All scripts & Asset Paths containing the `uMarketingSuite` keyword will need to be renamed to:
   * `Assets/Umbraco.Engage/Scripts/umbracoEngage.analytics.js`
   * `Assets/Umbraco.Engage/Scripts/umbracoEngage.ga4-bridge.js`
   * `Assets/Umbraco.Engage/Scripts/umbracoEngage.blockerdetection.js`
-* The Cockpit Partial View has been moved to:
+* The Cockpit Partial View has been moved, and needs to be changed to:
```
[the remark this answers is not in the document]
Comment (Jette94) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:134]: By replacing the uMarketingSuite Cockpit with Engage, the location will change - therefor you will need to change the path of the cockpit in the template :D

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md [hunk 2]
```diff
@@ -253,12 +257,12 @@ Run this script to migrate all the Umbraco data.
 
 ## Step 6: Finalize the migration
 
-1. Delete any `obj`/`bin` folders in your projects to ensure a clean build.
-2. Recompile all projects and ensure all dependencies are restored correctly.
-3. Contact Umbraco Support for a license key.
+1. Delete any `obj`/`bin` folders in your projects to ensure a clean build. 
+3. Recompile all projects and ensure all dependencies are restored correctly.
```
[the remark this answers is not in the document]
Comment (Jette94) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:261]: I'm not sure why it shows changes here

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sofietoft) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:134]: Alright! And which template is this exactly?
Just wondering if we should make that clearer, as currently it sounds like something needs to be moved to change a path 😅

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sofietoft) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:261]: OK. I'll revert 💪

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md [hunk 3]
```diff
@@ -63,6 +63,10 @@ The result should look like this:
 
 If any of these checks return a failure, please resolve the issue before proceeding with the migration.
 
+{% hint style="info" %}
```
[the remark this answers is not in the document]
Comment (sofietoft) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:66]: Will you make these changes, or should I? :)

**Review by sofietoft — COMMENTED** [review]
Thanks for considering the suggestions.

Did you see my suggestion to move the note about Cloud ABOVE the list?

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jette94) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:66]: Making them now :D

File: 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jette94) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:134]: It's the template for the page setup! If set up in a master template, that would be it. It needs to be in the bottom of the <body> tag

[the remark this answers is not in the document]
Comment (sofietoft) [reply 13/umbraco-engage/upgrading/migrate-from-umarketingsuite.md:134]: Alright. Should we do something like this instead then:

"The Cockpit partial view has been moved, and any references should be changed to:"

**Review by sofietoft — APPROVED** [review]

**Pull request merged**